src/Silindir.java

```diff
@@ -0,0 +1,44 @@
+public class Silindir extends Daire {
+
+    private double yukseklik;
+
+    private double yaricap;
```

Comment:
Daireyi extends ettiginiz için yaricap degerini burada tekrar tanıtmanıza gerek yok.